packages/.vitepress/config.ts

```diff
@@ -167,25 +170,145 @@ export default defineConfig({
     ['link', { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css2?family=Inter:wght@400;500;600&display=swap' }],
     ['link', { rel: 'stylesheet', href: 'https://fonts.googleapis.com/css2?family=Fira+Code&display=swap' }],
   ],
 
+  transformHead,
+  pwa: {
+    outDir: resolve(__dirname, 'dist'),
+    registerType: 'autoUpdate',
+    injectRegister: 'inline',
+    experimental: {
+      includeAllowlist: true,
+    },
+    manifest: {
+      id: '/',
+      name: 'VueUse',
+      short_name: 'VueUse',
+      description: 'Collection of Essential Vue Composition Utilities',
+      theme_color: '#ffffff',
+      start_url: '/',
+      lang: 'en-US',
+      dir: 'ltr',
+      orientation: 'natural',
+      display: 'standalone',
+      display_override: ['window-controls-overlay'],
+      categories: ['development', 'developer tools'],
+      icons: [
+        {
+          src: '/pwa-64x64.png',
+          sizes: '64x64',
+          type: 'image/png',
+        },
+        {
+          src: '/pwa-192x192.png',
+          sizes: '192x192',
+          type: 'image/png',
+        },
+        {
+          src: '/pwa-512x512.png',
+          sizes: '512x512',
+          type: 'image/png',
+          purpose: 'any',
+        },
+        {
+          src: '/maskable-icon.png',
+          sizes: '512x512',
+          type: 'image/png',
+          purpose: 'maskable',
+        },
+      ],
+      edge_side_panel: {
+        preferred_width: 480,
+      },
+      screenshots: [{
+        src: 'og.png',
+        sizes: '1281x641',
+        type: 'image/png',
+        label: `Screenshot of VueUse`,
+      }],
+    },
+    workbox: {
+      clientsClaim: true,
+      cleanupOutdatedCaches: true,
+      globPatterns: ['**/*.{css,js,html,svg,png,ico,txt,woff2}'],
+      globIgnores: ['og-*.png'],
+      runtimeCaching: [
+        {
+          urlPattern: /^https:\/\/fonts\.googleapis\.com\/.*/i,
+          handler: 'CacheFirst',
+          options: {
+            cacheName: 'google-fonts-cache',
+            expiration: {
+              maxEntries: 10,
+              maxAgeSeconds: 60 * 60 * 24 * 365, // <== 365 days
+            },
+            cacheableResponse: {
+              statuses: [0, 200],
+            },
+          },
+        },
+        {
+          urlPattern: /^https:\/\/fonts\.gstatic\.com\/.*/i,
+          handler: 'CacheFirst',
+          options: {
+            cacheName: 'gstatic-fonts-cache',
+            expiration: {
+              maxEntries: 10,
+              maxAgeSeconds: 60 * 60 * 24 * 365, // <== 365 days
+            },
+            cacheableResponse: {
+              statuses: [0, 200],
+            },
+          },
+        },
+        {
+          urlPattern: /^https:\/\/avatars\.githubusercontent\.com\/.*/i,
+          handler: 'NetworkFirst',
+          options: {
+            cacheName: 'avatars-images-cache',
+            expiration: {
+              maxEntries: 10,
```

Comment:
Increase this number

Reply:
I'm going to remove this handler, we've a lot of contributors and the avatars from gh, we're wasting space due to crossorigin and opaque padding